README.md

```diff
@@ -832,6 +832,19 @@ example.pendingMocks() // []
 
 example.get("/pathB").optionally().reply(200);
 example.pendingMocks() // []
+
+// You can also pass a boolean argument to `optionally()`. This
+// is useful if you want to conditionally make a mocked request
+// optional.
+var getMock = function(optional) {
+  return example.get("/pathC").optionally(optional).reply(200);
+}
+
+getMock(true);
+example.pendingMocks() // []
+getMock(false);
+example.pendingMocks() // ["GET http://example.com:80/pathC"]
```

Comment:
I wonder if we need the `getMock` function? Wouldn’t that be enough?

```js
example.get("/pathC").optionally(true).reply(200);
example.pendingMocks() // []
example.get("/pathC").optionally(false).reply(200);
example.pendingMocks() // ["GET http://example.com:80/pathC"]
```

Reply:
I'm happy to be led by you on this.

I included the function because I think it makes clearer *why* you'd want to do this. I think just passing in the `true` and `false` might look a bit odd (i.e. "why do I need this parameter when for the mock to be required is the default?").